src/misc/schemas/binarySchema.ts

```diff
@@ -1,5 +1,5 @@
 import { z } from 'zod';
 
-export const binarySchema = z.string().regex(/\b[01]+\b/, {
+export const binarySchema = z.string().regex(/^[01]*$/, {
```

Comment:
are there any tests to cover this change?

Reply:
It's basically a more concise way to write the same regexp, nothing really changed, but if smth. would go wrong, there are tests against it.